app/footer.tsx

```diff
@@ -1,3 +1,180 @@
+            </div>
+            <div className="pr-5">
+              <Image src={mail} alt="" className="" />
+            </div>
+            <div className="pr-5">
+              <Image src={lang} alt="" className="" />
+            </div>
+            <div className="pr-5">
+              <Image src={distance} alt="" className="" />
+            </div>
+          </div>
+        </div>
+
+        <div className="sm:items-center flex lg:items-center justify-center origin-center rotate-90 lg:rotate-0 ">
+          <Image src={footerimg} className="h-[362px] w-[74px] " alt="" />
+        </div>
+
+        <div className="flex lg:items-center justify-center lg:flex-row  flex-col flex-wrap items-start ">
+          <div className="lg:px-10  px-5 ">
+            <ul>
+              <li className="font-bold md:text-[24px] py-3 sm:text-[16px]">
+                Quick Links
+              </li>
+              {quickLinks.map((item, index) => (
+                <li
+                  key={index}
+                  className="font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]"
+                >
+                  <a href={item.link}>{item.text}</a>
+                </li>
+              ))}
+            </ul>
+          </div>
+          <div className="lg:px-10  pb-8 px-5">
+            <ul>
+              <li className="font-bold md:text-[24px] py-3 sm:text-[16px]">
+                About Us
+              </li>
+              {aboutUsItems.map((item, index) => (
+                <li
+                  key={index}
+                  className="font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]"
+                >
+                  <a href={item.link}>{item.text}</a>
+                </li>
+              ))}
+            </ul>
+          </div>
+          <div className="lg:px-10 pb-8 px-5">
+            <ul>
+              <li className="font-bold md:text-[24px] py-3 sm:text-[16px]">
+                Departments
+              </li>
+              {departmentsItems.map((item, index) => (
+                <li
+                  key={index}
+                  className="font-medium text-[#8B949E] py-1 md:text-[20px] sm:text-[12px]"
+                >
+                  <a href={item.link}>{item.text}</a>
+                </li>
+              ))}
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div className="flex items-center justify-between bg-[#101010]">
+        <div className=" py-5 lg:pl-20 sm:pl-10 font-inter text-[#5A6475]">
+          <ul className="flex items-center justify-center lg:flex-row flex-col">
+            <div className="flex flex-row">
+              <li className="px-3">© 2022 NIT KKR, Inc.</li>
+              <li className="px-3">Terms</li>
+            </div>
+            <li className="pl-3">Privacy (Updated 08/2022)</li>
+          </ul>
+        </div>
+
+        <div className=" flex   flex-row  items-center justify-center flex-shrink-0 py-5  lg:pr-10 sm:pr-4 ">
```

Comment:
Please format and remove these issues
```suggestion
        <div className="flex flex-row items-center justify-center flex-shrink-0 py-5 lg:pr-10 sm:pr-4">
```